schemas/file.yml

```
@@ -4,169 +4,178 @@
+      type: keyword
+      description: Target path for symlinks.
+      multi_fields:
+        - type: text
+          name: text
+
+    - name: extension
+      level: extended
+      type: keyword
+      description: File extension.
+      example: png
+
+    - name: type
+      level: extended
+      type: keyword
+      description: File type (file, dir, or symlink).
+      example: file
+
+    - name: device
+      level: extended
+      type: keyword
+      description: Device that is the source of the file.
+      example: sda
+
+    - name: inode
+      level: extended
+      type: keyword
+      description: Inode representing the file in the filesystem.
+      example: "256383"
+
+    - name: uid
+      level: extended
+      type: keyword
+      description: >
+        The user ID (UID) or security identifier (SID) of the file owner.
+      example: "1001"
+
+    - name: owner
+      level: extended
+      type: keyword
+      description: File owner's username.
+      example: alice
+
+    - name: gid
+      level: extended
+      type: keyword
+      description: Primary group ID (GID) of the file.
+      example: "1001"
+
+    - name: group
+      level: extended
+      type: keyword
+      description: Primary group name of the file.
+      example: alice
+
+    - name: mode
+      level: extended
+      type: keyword
+      description: Mode of the file in octal representation.
+      example: "0640"
+
+    - name: size
+      level: extended
+      type: long
+      short: File size in bytes.
+      description: >
+        File size in bytes.
+
+        Only relevant when `file.type` is "file".
+      example: 16384
+
+    - name: mtime
+      level: extended
+      type: date
+      description: Last time the file content was modified.
+
+    - name: ctime
+      level: extended
+      type: date
+      short: Last time the file attributes or metadata changed.
+      description: >
+        Last time the file attributes or metadata changed.
+
+        Note that changes to the file content will update `mtime`. This implies
+        `ctime` will be adjusted at the same time, since `mtime` is an attribute
+        of the file.
+
+    - name: created
+      level: extended
+      type: date
+      short: File creation time.
+      description: >
+        File creation time.
+
+        Note that not all filesystems store the creation time.
+
+    - name: accessed
+      level: extended
+      type: date
+      short: Last time the file was accessed.
+      description: >
+        Last time the file was accessed.
+
+        Note that not all filesystems keep track of access time.
+
+    - name: mime_type
+      level: extended
+      type: keyword
+      short: Media type of file, document, or arrangement of bytes.
+      description: >
+        MIME type should identify the format of the file or stream of bytes using
+        [IANA official types](https://www.iana.org/assignments/media-types/media-types.xhtml),
```

Please replace the Markdown link with an asciidoc link.

```suggestion
        https://www.iana.org/assignments/media-types/media-types.xhtml[IANA official types],
```

We even normally try to avoid asciidoc link, as the descriptions are used elsewhere as well. But this is one case where the link is just too long for it to look good without a real link :-)